SCClassLibrary/SCDoc/SCDocRenderer.sc

```diff
@@ -119,17 +120,34 @@ SCDocHTMLRenderer {
 			baseDir = baseDir ++ "/..";
 		};
 
+		thisIsTheMainHelpFile = (doc.title=="Help") and:
+			{((thisProcess.platform.name===\windows) and: (folder=="Help")) or: {folder==""}};
+
+		stream
+		<< "<!doctype html>"
+		<< "<html lang='en'>"
```

Comment:
i don't think its worth the time to fix the scdoc parser to add support ofr language metadata.[html5 wants you to always have the language attr,](https://www.w3.org/International/questions/qa-html-language-declarations) so i can't just leave it out.